conf/map_darkstar.conf

```diff
@@ -199,3 +199,9 @@ healing_tick_delay: 10
 #Central message server settings (ensure these are the same on both all map servers and the central (lobby) server
 msg_server_port: 54003
 msg_server_ip: 127.0.0.1
+
+#Set to 1 to enable server side anti-cheating measurements
+anticheat_enabled: 1
+#Set to change the cheating severity level that will cause players to be automatically jailed.
+#Defaults to 4 (severe). Set to 0 to disable auto-jailing.
+#anticheat_jail_threshold: 4

```

Comment:
Should prolly default these off for now, but we'll need the team weigh in on the system in general. May be awhile, everybody has been busy with IRL stuff lately. Thank you in advance for your patience.

Reply:
OK, I'll disable it once the CI tests end (pass or fail). Apparently I messed up some makefiles so I had to push again anyway.
@TeoTwawki I may need some assistance if they don't pass, I compile on Visual Studio so makefiles are different. Sorry about that.

Reply:
OK, disabled by default. Makefiles also seem to work (CI tests passed).